tests/core/services/image_manipulation/test_image_manipulation.py

```diff
@@ -30,6 +32,7 @@ def tearDown(self) -> None:
     def test_save_image_to_buffer(self):
         img_to_save = MagicMock()
         img_to_save.save = MagicMock()
+        ImageOps.exif_transpose = MagicMock(return_value=img_to_save)
```

Comment:
This is not enough, is it too complicated to create a test about it?

Reply:
This is **not** a test for the transpose function, this is merely an update to old test that verified the save function. We do not  (and should not) have unit test to verify something that PIL does. If we want to have that we should have an integration test that will check the image input and output

Reply:
I would start to do some integration tests and start testing this behavior and in the future we will add more!